zebrad/src/components/mempool/storage/eviction_list.rs

```diff
@@ -0,0 +1,107 @@
+//! [`EvictionList`] represents the transaction eviction list with
+//! efficient operations.
+use std::{
+    collections::{HashMap, VecDeque},
+    time::{Duration, Instant},
+};
+
+use zebra_chain::transaction;
+
+/// An eviction list that allows to efficiently add entries, get entries,
+/// and remove older entries in the order they were inserted.
+pub struct EvictionList {
+    // Maps each TXID In the list to the instant they were added.
```

Comment:
Docs: clarification, typo

```suggestion
    // Maps each TXID in the list to the most recent instant they were added.
```